### DISCUSSION TITLE
关于在宝塔搭建翼龙面板的问题反馈

### DISCUSSION THREAD

**Discussion by jgc9527** [ilworkcn/pterodactyl-chinese#18] [General]
搭建完翼龙之后需要更改自己需要的预设组
但是我在上传预设组文件时上传的文件为别的权限（指755权限以外）导致页面出现500错误，无法上传预设文件

**Comment by vlssu** [comment]
> 搭建完翼龙之后需要更改自己需要的预设组 但是我在上传预设组文件时上传的文件为别的权限（指755权限以外）导致页面出现500错误，无法上传预设文件

下载预设的仓库时，你需要全部下载存档文件。然后再解压后上传。
